py/openage/codegen/codegen.py

```diff
@@ -6,7 +6,7 @@
 
 from openage.log import dbg
 
-contenttemplate = """\
+contenttemplate = """
```

Comment:
for the content template, the `\` is unfortunately necessary; otherwise the generated files will start with a newline (where a copyright line would be expected)